### DISCUSSION TITLE
Iota tokens gone

### DISCUSSION THREAD

**Discussion by dariuszuba** [iotaledger/firefly#8184] [Help & Support] [answered]
I opened my Firefly profile after not being on it for a while.
The Iota tokens I had on my wallet are not visible there any more.

There is only visible some tokens which are coming to my wallet from vesting.
I don't remember what vesting is, and there is no visible history if my tokens were invested.

Does anyone know how to find out where my Iota tokens went?
Thank you.

**Comment by tackar66** [comment]
Hallo,
bei mir leider das gleiche, 
hattest du auch eine Aktualisierung der Firefly Wallet gemacht? 
Ich habe eine gemacht bevor ich die Wallet geöffnet habe, seit dem werden meine Token nicht mehr angezeigt.
Gruß

**Comment by begonaalvarezd** [comment, marked as answer]
You can run the balance finder from settings > advanced to find missing balances. 

To explain about the vesting rewards and other changes, here is some quick information about ASMB and SMR:
- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app because your rewards are in the shimmer network, not the iota network. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

**Reply by Mark7171** [reply]
Thank you very much, I hadn't logged in for a while either and only found wallet 1, while wallets 2-5 were missing. Almost had a heart attack. Now everything is back thanks to several searches for missing balances.
